internal/controlplane/handlers_oauth.go

```diff
@@ -186,6 +191,40 @@ func (s *Server) GetAuthorizationURL(ctx context.Context,
 	}, nil
 }
 
+func (s *Server) alllowedRedirectURL(redirectUrl *url.URL) bool {
+	if redirectUrl == nil || redirectUrl.String() == "" {
+		return true // Empty URL is allowed
+	}
+	if redirectUrl.Host == "localhost" {
```

Comment:
https://pkg.go.dev/net/url#URL documents `Host` as being "host or host:port (see Hostname and Port methods).